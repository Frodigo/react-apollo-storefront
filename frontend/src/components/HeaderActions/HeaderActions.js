import React from 'react';
import { Nav } from 'react-bootstrap';

const HeaderActions = () => {
    return (
        <Nav style={{marginLeft: 'auto'}}>
            <Nav.Item>
                <Nav.Link href="#">
                    <i className="bi bi-heart-fill"></i>
                </Nav.Link>
            </Nav.Item>
            <Nav.Item>
                <Nav.Link href="#">
                    <i className="bi bi-person-fill"></i>
                </Nav.Link>
            </Nav.Item>
            <Nav.Item>
                <Nav.Link href="#">
                    <i className="bi bi-cart-fill"></i>
                </Nav.Link>
            </Nav.Item>
        </Nav>
    );
};


export default HeaderActions;
