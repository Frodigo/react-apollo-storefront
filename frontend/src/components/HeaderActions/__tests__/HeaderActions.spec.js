import React from 'react';
import { render } from '@testing-library/react';

import HeaderActions from '../HeaderActions';

test('HeaderActions renders correctly', () => {

    const { container } = render(<HeaderActions />);

    expect(container).toMatchSnapshot();
});

