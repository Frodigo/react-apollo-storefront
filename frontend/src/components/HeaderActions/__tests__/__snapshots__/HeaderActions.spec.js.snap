// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderActions renders correctly 1`] = `
<div>
  <div
    class="nav"
  >
    <div
      class="nav-item"
    >
      <a
        class="nav-link"
        data-rb-event-key="#"
        href="#"
        role="button"
      >
        <i
          class="bi bi-heart-fill"
        />
      </a>
    </div>
    <div
      class="nav-item"
    >
      <a
        class="nav-link"
        data-rb-event-key="#"
        href="#"
        role="button"
      >
        <i
          class="bi bi-person-fill"
        />
      </a>
    </div>
    <div
      class="nav-item"
    >
      <a
        class="nav-link"
        data-rb-event-key="#"
        href="#"
        role="button"
      >
        <i
          class="bi bi-cart-fill"
        />
      </a>
    </div>
  </div>
</div>
`;
